packages/@webex/helper-image/test/unit/spec/index.js

```diff
@@ -3,7 +3,8 @@
  */
 
 import {assert} from '@webex/test-helper-chai';
-import {readExifData, orient, updateImageOrientation} from '@webex/helper-image';
+import {readExifData, updateImageOrientation} from '@webex/helper-image';
```

Comment:
I think its ok to refer to the package in the unit test link this 

but the orient file can also be referred using package name 

cc @InteractiveTimmy  correct me if I am wrong

Reply:
You mean like this:
`import {orient} from '@webex/helper-image/src/orient';`

Reply:
still little confused on why orient was moved out to a file